<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPeriod</class>
 <widget class="QDialog" name="DlgPeriod">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>418</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>418</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>418</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="deStart">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbStartDate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>32</y>
     <width>60</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbUser">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>6</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Guest</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>62</y>
     <width>60</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Height, sm</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbHeight">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="minimum">
    <number>50</number>
   </property>
   <property name="maximum">
    <number>250</number>
   </property>
   <property name="value">
    <number>170</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sbWeight">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>20.000000000000000</double>
   </property>
   <property name="maximum">
    <double>300.000000000000000</double>
   </property>
   <property name="value">
    <double>80.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>92</y>
     <width>60</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Weight, kg</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>122</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Activity</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbActivity">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Sedentary lifestyle</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Small (sports 1-3 days a week)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Medium (sport 3-5 days a week)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>High (sport 6-7 days a week) </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Very high (physical work, 2 workouts per day</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>158</y>
     <width>320</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; color:#0000ff;&quot;&gt;Ratio of protein, fat, carbohydrates to achieve the goal:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>182</y>
     <width>64</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Proteins, %</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>272</y>
     <width>71</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Variation, %</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>212</y>
     <width>64</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Fats, %</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>242</y>
     <width>97</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Carbohydrates, %</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbProt">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>20</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbFat">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>20</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbCarb">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>20</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sbVariation">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="value">
    <double>2.500000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Note</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="teNote">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgPeriod</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgPeriod</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
